builders/bower.go

```diff
@@ -141,7 +142,48 @@ func (builder *BowerBuilder) IsModule(target string) (bool, error) {
 	return false, errors.New("IsModule is not implemented for BowerBuilder")
 }
 
-// InferModule is not implemented
-func (builder *BowerBuilder) InferModule(target string) (module.Module, error) {
-	return module.Module{}, errors.New("InferModule is not implemented for BowerBuilder")
+// DiscoverModules is not implemented
+func (builder *BowerBuilder) DiscoverModules(dir string) ([]config.ModuleConfig, error) {
+	moduleConfigs := []config.ModuleConfig{}
```

Comment:
You should use the `var slice []T` form unless the length is known, in which case use `make`.